src/Microsoft.Health.Fhir.Core/Features/Operations/Validate/ValidateResourceOperationValidator.cs

```diff
@@ -10,13 +10,13 @@
 namespace Microsoft.Health.Fhir.Core.Messages.Operation
 {
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1710:Identifiers should have correct suffix", Justification = "Follows validator naming convention.")]
```

Comment:
>SuppressMessage [](http://example.com/codeflow?start=37&length=15)

Is this attribute suppression still needed?

Reply:
No we don't! Removed